ignite/contrib/handlers/throughput_benchmark_wrapper.py

```diff
@@ -0,0 +1,216 @@
+import torch
+import torch.jit
+import contextlib
+from typing import Callable, Union
+from ignite.engine import Events, Engine
+from torch.utils import ThroughputBenchmark
```

Comment:
Can you wrap this in a way, that you first check the pytorch version if it is recent enough? This will probably result in a more understandable error message then a plain import error

Reply:
Are you only referring to `from torch.utils`?
Should I simply use a `try except(ImportError)`  to get the ThroughputBenchmark, or should I inspect the `torch.__version__` and raise a different error, if it is too low?

In the other contrib files I've seen that the specific module is loaded lazily and then a runtime error is raised if it doesn't exist. I could also do that, but then I couldn't annotate the return of `def stats`.

Reply:
I've created a typed stub for `ExecutionStats` so that I don't have to import it manually.
The check if I can access `torch.utils.ThroughputBenchmark` is now done lazily and is more similar to the other approaches.